rest-api.md

```diff
@@ -521,6 +521,7 @@ startTime | LONG | NO |
 endTime | LONG | NO |
 
 * If startTime and endTime are not sent, the most recent klines are returned.
+* If startTime is provided then endTime is also manditory.
```

Comment:
> manditory

Reply:
FYI spelling error: should be "mandatory".